app/models/page.rb

```diff
@@ -38,7 +38,7 @@ def body_present
   end
 
   def unique_slug_including_users_and_orgs
-    slug_exists = User.exists?(username: slug) || Organization.exists?(slug: slug) || Podcast.exists?(slug: slug)
+    slug_exists = User.exists?(username: slug) || Organization.exists?(slug: slug) || Podcast.exists?(slug: slug) || slug.include?("sitemap-")
```

Comment:
If we create other partial match restrictions like this we can abstract this, but I feel like rule of three works here.